src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -85,7 +85,19 @@ mocha.describe('readings API', () => {
 					expectCompareToEqualExpected(res, expected);
 				});
 
-				// Add C6 here
+				mocha.it('C6: 28 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as kWh', async () => {
```

Comment:
The text in the design doc is slightly different with "28 day shift end 2022-10-31 17:12:34 (partial hour) for 15 minute reading intervals and quantity units & kWh as kWh". Could this be updated to match?

Reply:
Will do, thank you for your prompt feedback!